src/lib/task_listener.js

```diff
@@ -61,8 +62,8 @@ export default class TaskListener extends EventEmitter {
               }
               this.cleanupRunningState(state);
             }
-          }();
-        }.bind(this)
+          })();
+        }).bind(this)
```

Comment:
This `.bind` isn't needed for arrow functions since they don't have their own context.